### PR TITLE
[HttpKernel] Add parameters kernel.runtime_mode and kernel.runtime_mode.*, all set from env var APP_RUNTIME_MODE

### LINKED ISSUE #19066: [HttpKernel] Add parameters `kernel.runtime_mode` and `kernel.runtime_m…
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#52079
| PR author(s) | @nicolas-grekas
| Merged in    | 6.4

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 6.4 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/symfony/symfony-docs/issues/19066

### CODE REVIEW

File: reference/configuration/kernel.rst [hunk 1]
```diff
@@ -308,6 +308,56 @@ the configuration options used to run the application. This allows for example
...
+
+    The ``kernel.runtime_mode`` parameter was introduced in Symfony 6.4.
+
+``kernel.runtime_mode.cli``
+---------------------------
+
+**type**: ``boolean`` **default**: ``%env(not:default:kernel.runtime_mode.web:)%``
+
+Whether the application is running in a CLI environment. By default,
+this value is the opposite of the ``kernel.runtime_mode.web`` parameter.
```
Comment (alamirault) [inline reference/configuration/kernel.rst]: As we talk about `kernel.runtime_mode.web` here, what do you think to set `kernel.runtime_mode.web` above `cli ?`

Comment (alexandre-daubois) [reply reference/configuration/kernel.rst]: I sorted alphabetically but I think putting web above cli makes more sense indeed!

**Review by alamirault — APPROVED** [review]

File: reference/configuration/kernel.rst [hunk 2]
```diff
@@ -308,6 +308,56 @@ the configuration options used to run the application. This allows for example
...
+``APP_RUNTIME_MODE`` env var.
+
+.. versionadded:: 6.4
+
+    The ``kernel.runtime_mode`` parameter was introduced in Symfony 6.4.
+
+``kernel.runtime_mode.web``
+---------------------------
+
+**type**: ``boolean`` **default**: ``%env(bool:default::key:web:default:kernel.runtime_mode:)%``
```
Comment (OskarStark) [inline reference/configuration/kernel.rst:329]:
```suggestion
**type**: ``boolean`` **default**: ``%env(bool:default::key:web:default:kernel.runtime_mode??)%``
```

Comment (OskarStark) [reply reference/configuration/kernel.rst:329]: Is this correct?

Comment (alexandre-daubois) [reply reference/configuration/kernel.rst:329]: I took this from https://github.com/symfony/symfony/blob/6.4/src/Symfony/Component/HttpKernel/Kernel.php#L574

This new syntax was introduced here if I am right: https://github.com/symfony/symfony/pull/52079/files#r1363574735

**Comment by OskarStark** [comment]
Thank you Alexandre.

**Pull request merged**